mempool/txpool.go

```diff
@@ -107,6 +107,36 @@ func (mp *TxPool) GetTxsInPool() []*Transaction {
 	return txs
 }
 
+func (mp *TxPool) GetTxDetailsInPool() ([]*Transaction, []*Transaction, []*Transaction, []string, []string, []Uint256) {
```

Comment:
Define a Snapshot struct to instead of returning so many arguments.
And change method name to GetMemPoolSnapshot().

Reply:
Good idea, I'll fix it